projects/limble-tree/src/lib/core/tree-branch/tree-branch.ts

```diff
@@ -83,23 +108,43 @@ export class TreeBranch<UserlandComponent>
       this.dispatch(new DestructionEvent(this));
    }
 
+   /** Run Angular change detection on this branch */
    public detectChanges(): void {
       this.branchController.detectChanges();
    }
 
+   /**
+    * Emits the specified TreeEvent.
+    *
+    * Caution: It is not recommended to manually emit TreeEvents that are already
+    * provided by the library. For example, it is not recommended to emit a
+    * `GraftEvent`, `DestructionEvent`, etc. These events may be used by the tree,
+    * and emitting them manually may cause unexpected behavior. Instead, we
+    * recommend implementing the TreeEvent interface with your own custom events
+    * and dispatching those.
+    */
```

Comment:
nit: This public function has a parameter, and you can use `@param` here to specify the purpose of the param.

Something like `@param event - The {@link TreeEvent} that will be emitted.`